nodejs/week1/homework/src/backend/app.js

```diff
@@ -0,0 +1,65 @@
+const express = require("express");
+const app = express();
+
+// import data here
+
+const meals = require("./data/meals");
+const reviews = require("./data/reviews.json");
+const reservations = require("./data/reservations.json");
+
+// this is where you will be adding your routes
+
+
+const random = (arr) => arr[Math.floor(Math.random() * arr.length)];
+
+const mealsWithReviews = meals.map((meal) => {
+  return {
+    id: meal.id,
+    maxNumberOfGuests: meal.maxNumberOfGuests,
+    title: meal.title,
+    description: meal.description,
+    time: 12 + meal.id,
+    address: "asd " + meal.id,
+    price: meal.price,
+    reviews: reviews.filter((review) => review.mealId === meal.id),
+    createdAt: meal.createdAt,
+  };
+});
+console.log(mealsWithReviews);
+
+app.get("/", async (request, response) => {
+  response.send("Meal Sharing Web App");
```

Comment:
In this case both .send and .json work but it's preferred to use .json -> https://hackyourfuture-cph.slack.com/archives/C02EXR39E5R/p1645710223738659?thread_ts=1645709966.699389&cid=C02EXR39E5R

Reply:
That was interesting ,thanks 💙.I have just implemented it .